ts/kpt-functions/src/io.ts

```diff
@@ -26,7 +26,7 @@ export enum FileFormat {
   JSON,
 }
 type FilePath = string;
-const YAML_STYLE: DumpOptions = {
+export const YAML_STYLE: DumpOptions = {
```

Comment:
in general functions shouldn't care YAML style. kpt CLI is the orchestrator and should run equivalent of 'kpt fmt' on the output.